### PR TITLE
Test beautiful soup

### PR DESCRIPTION
**Pull request by nnaeyno**
testing Beautiful soup api and how it would work for our web page

### CODE REVIEW

File: scrapping.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    def get_soup(self, url):
+        """Fetches a webpage and returns a BeautifulSoup object."""
+        response = requests.get(url)
+        response.raise_for_status()  # Raise an exception for bad responses
+        soup = BeautifulSoup(response.text, 'html.parser')
+        return soup
+
+    def scrape_receptebi(self):
+        # Step 1: Load the main page to find the "/receptebi/" link
+        main_page_url = f"{self.base_url}/"
```
Comment (lizekss) [inline scrapping.py:20]: does it need the trailing slash?

File: scrapping.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        wvnianebi_link = recipe_nav_body.find('a', href="/receptebi/cat/wvnianebi/")['href']
+        print(wvnianebi_link)
+        # Step 4: Follow the "/wvnianebi/" link
+        wvnianebi_url = self.base_url + wvnianebi_link
+        wvnianebi_soup = self.get_soup(wvnianebi_url)
+
+        # You can now scrape the content of the "/wvnianebi/" page as needed
+        # For example, let's print all recipe titles available on this page
+
+        recipe_nav_body = wvnianebi_soup.find('div', class_='recipe__nav-body')
```
Comment (lizekss) [inline scrapping.py:42]: what does this nav body mean? is it some class specific to the website?

File: scrapping.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import requests
+from bs4 import BeautifulSoup
+import os
+
+
+class BS4Scrapping():
+    # Base URL of the website
```
Comment (lizekss) [inline scrapping.py]: is this comment redundant?

File: scrapping.py (hunk 2 above)
Comment (nnaeyno) [reply scrapping.py:42]: yes 
i was checking the html of the page

**Pull request merged**